### PR TITLE
feat: improve error message on missing user data

### PR DESCRIPTION
**Pull request by hsiliev**
Replaces 
```
Cannot invoke "net.atos.zac.authentication.LoggedInUser.getId()" because "loggedInUser" is null: java.lang.NullPointerException: Cannot invoke "net.atos.zac.authentication.LoggedInUser.getId()" because "loggedInUser" is null
```

with: 
```
java.lang.NullPointerException: No logged in user found. Please ensure that user did not log out and its session is still active
```

Solves: PZ-2487

### CODE REVIEW

File: src/main/java/net/atos/zac/policy/input/DocumentInput.java [hunk 1]
```diff
@@ -14,7 +15,7 @@ public class DocumentInput extends UserInput {
     @JsonbProperty("document")
     private final DocumentData documentData;
 
-    public DocumentInput(final LoggedInUser loggedInUser, final DocumentData documentData) {
+    public DocumentInput(@NotNull LoggedInUser loggedInUser, final DocumentData documentData) {
```
Comment (edgarvonk) [inline src/main/java/net/atos/zac/policy/input/DocumentInput.java:18]: Can still be final no?



```suggestion
    public DocumentInput(@NotNull final LoggedInUser loggedInUser, final DocumentData documentData) {
```

Comment (edgarvonk) [inline src/main/java/net/atos/zac/policy/input/DocumentInput.java:18]: Do these @NotNull annotations really work here? I thought they only worked with a related @Valid annotation somewhere earlier in de callstack?

Comment (hsiliev) [reply src/main/java/net/atos/zac/policy/input/DocumentInput.java:18]: Nope - totally not working. These are left from my first attempts. Will get rid of these.

Comment (hsiliev) [reply src/main/java/net/atos/zac/policy/input/DocumentInput.java]: Got rid of these

**Review by edgarvonk — APPROVED** [review]

**Pull request merged**